src/main/g8/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '2.2'
```

Comment:
This file is convenience for development. I'd rather move it into a specific folder (e.g. `dev-tools/` or something else).

Reply:
I guess there is a "standard" for that. Will docker-compose find it even if it's in a sub-directory?

Reply:
I have no idea 😅